nservicebus/pipeline/manipulate-with-behaviors_registernew_core_[6.0,).partial.md

```diff
@@ -5,4 +5,6 @@ snippet:RegisterBehaviorEndpointConfiguration
 
 Behaviors can also be registered from a `Feature` as shown below:
 
-snippet:RegisterBehaviorFromFeature
\ No newline at end of file
+snippet:RegisterBehaviorFromFeature
+
+WARNING: Behaviors are only created once and the same instance is reused on every invocation of the pipeline, even when registering the behavior to be created by a Dependency Injection Container. Never use dependency injection for dependencies configured with an *InstancePerUnitOfWork* scope.
```

Comment:
Hm I don't think that statement holds up. E.g. you can define a stateless dependency in transient/instancePerCall scope and that works just fine.